Chandra/cmd_states/cmd_states.py

```diff
@@ -393,14 +399,16 @@ def get_state0(date=None, db=None, date_margin=10, datepar='datestop'):
         db = Ska.DBI.DBI(dbi='sybase', server='sybase', user='aca_read',
                          database='aca')
 
-    # Date for which cmd_states are certainly reliable
-    definitive_date = DateTime(time.time() - date_margin * 86400.,
-                               format='unix').date
-    if date is None or date > definitive_date:
-        date = definitive_date
-    else:
+    if date is not None:
```

Comment:
Obviously in practice your version would work out OK, but for this exercise I was aiming for consistency.